system/doc/reference_manual/ref_man_records.md

```diff
@@ -55,6 +54,29 @@ definition is placed in an include file.
 > using the syntax described in this section. In earlier releases, it was
 > necessary to use the `m:shell` built-in function `rd/2`.
 
+> #### Change {: .info }
+>
+> In Erlang/OTP 28, variables are allowed in the record definition.
+> If a record definition contains a variable, a function call will be made to
+> initialize the records omitted fields.
```

Comment:
Isn't the function call an implementation detail? Why should it be mentioned? I think that it would be to describe the semantics.

```suggestion
> If a record definition contains a variable, the compiler will insert a function call to
> initialize the record's omitted fields.
```